ehr_billing/api-src/org/labkey/api/ehr_billing/pipeline/InvoicedItemsProcessingService.java

```diff
@@ -58,4 +67,26 @@ static InvoicedItemsProcessingService get()
      * @param invoiceId invoice Run Id for which additional processing can be performed
      */
     void performAdditionalProcessing(String invoiceId, User user, Container container);
+
+    default Pair<String,String> verifyBillingRunPeriod(User user, Container container, Date startDate, Date endDate) throws PipelineJobException
```

Comment:
What is this returning? Please add some JavaDoc, and indicate if it's @Nullable or @NotNull